presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/BenchmarkSTContains.java

```diff
@@ -0,0 +1,128 @@
+package com.facebook.presto.plugin.geospatial;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import com.esri.core.geometry.ogc.OGCPoint;
+import com.facebook.presto.geospatial.GeometryUtils;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.VerboseMode;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@State(Scope.Thread)
+@Fork(2)
+@Warmup(iterations = 5, time = 500, timeUnit = TimeUnit.MILLISECONDS)
```

Comment:
Average error seems rather high, e.g:
```
BenchmarkSTContains.stContainsInnerPointDeserialized               avgt   10   0.093 ±  0.045  ms/op
...
BenchmarkSTContains.stContainsOuterPointInEnvelopeDeserialized     avgt   10   0.094 ±  0.047  ms/op
```
consider more iterations.

Reply:
@sopel39 , thanks for the suggestion. With 10 iterations the error is very small:

```
Benchmark                                                          Mode  Cnt   Score    Error  Units
BenchmarkSTContains.deserialize                                    avgt   20   0.131 ±  0.004  ms/op
BenchmarkSTContains.deserializeEnvelope                            avgt   20  ≈ 10⁻⁴           ms/op
BenchmarkSTContains.stContainsInnerPoint                           avgt   20   0.205 ±  0.008  ms/op
BenchmarkSTContains.stContainsInnerPointDeserialized               avgt   20   0.064 ±  0.001  ms/op
BenchmarkSTContains.stContainsOuterPointInEnvelope                 avgt   20   0.210 ±  0.007  ms/op
BenchmarkSTContains.stContainsOuterPointInEnvelopeDeserialized     avgt   20   0.066 ±  0.002  ms/op
BenchmarkSTContains.stContainsOuterPointNotInEnvelope              avgt   20  ≈ 10⁻⁴           ms/op
BenchmarkSTContains.stContainsOuterPointNotInEnvelopeDeserialized  avgt   20  ≈ 10⁻⁵           ms/op
```